src/main/java/levy/fallingsand/Sand.java

```diff
@@ -52,28 +58,41 @@ public void fall() {
                     int direction1 = rightFirst ? +1 : -1;
                     int direction2 = rightFirst ? -1 : +1;
 
-                    if (field[y + 1][x + direction1] == 0) {
-                        field[y][x] = 0;
-                        field[y + 1][x + direction1] = 1;
-                    } else if (field[y + 1][x + direction2] == 0) {
-                        field[y][x] = 0;
-                        field[y + 1][x + direction2] = 1;
+                    // left edge
+                    if (x == 0) {
+                        if (field[y + 1][x] == 0) {
+                            field[y][x] = 0;
+                            field[y + 1][x] = 1;
+                        }
+                    }
```

Comment:
What does this code do from line 62-67?

Reply:
checks the left edge to see if anything there if nothing below it falls if something is below it stays

Reply:
If you're on the left edge, you should be able to fall to the right. Not just down.